src/main/java/org/folio/oaipmh/helpers/InstancesStorageHelper.java

```diff
@@ -40,4 +40,11 @@ static InstancesStorageHelper getStorageHelper() {
    * @return endpoint
    */
   String buildItemsEndpoint(Request request) throws UnsupportedEncodingException;
+
+  /**
+   * Gets endpoint to search for an instance by identifier
+   * @param id instance identifier
+   * @return endpoint to search for an instance by identifier
+   */
+  String getInstanceEndpoint(String id);
```

Comment:
I have to add separate method to search for instance instead of re-using `buildItemsEndpoint(...)` because we have to extract local storage identifier from passed identifier (which is in oai-identifier structure).